internal/builtins/mongodb.ts

```diff
@@ -8,85 +8,96 @@ export type DocumentOutput = Record<string, unknown>;
 export const find = async (
   mongodbResource: string,
   collection: string,
-  filter: Record<string, unknown> | undefined | null = null,
-  projection: Record<string, unknown> | undefined | null = null,
-  sort: Record<string, unknown> | undefined | null = null,
-  skip: number | undefined | null = null,
-  limit: number | undefined | null = null,
-  opts?: ClientOptions
+  opts: {
+    filter?: Record<string, unknown> | null;
+    projection?: Record<string, unknown> | null;
+    sort?: Record<string, unknown> | null;
+    skip?: number | null;
+    limit?: number | null;
+    client?: ClientOptions;
+  } = {}
 ): Promise<Run<ParamValues, DocumentOutput[] | undefined | null>> => {
+  const { filter, projection, sort, skip, limit, client } = opts;
   return getRuntime().execute(
     "airplane:mongodb_find",
     { collection, filter, projection, sort, skip, limit },
     { db: convertResourceAliasToID(mongodbResource) },
-    opts
+    client
   );
 };
 
+export type FindOptions = {
+  filter?: Record<string, unknown> | null;
+  projection?: Record<string, unknown> | null;
+  sort?: Record<string, unknown> | null;
+  client?: ClientOptions;
+};
+
+export type UpdateOptions = {
+  filter?: Record<string, unknown> | null;
+  upsert?: boolean | null;
+  client?: ClientOptions;
+};
+
 export const findOne = async (
   mongodbResource: string,
   collection: string,
-  filter: Record<string, unknown> | undefined | null = null,
-  projection: Record<string, unknown> | undefined | null = null,
-  sort: Record<string, unknown> | undefined | null = null,
-  opts?: ClientOptions
+  opts: FindOptions = {}
 ): Promise<Run<ParamValues, DocumentOutput | undefined | null>> => {
+  const { filter, projection, sort, client } = opts;
   return getRuntime().execute(
     "airplane:mongodb_findOne",
```

Comment:
Hmm, unrelated to your change, but this looks like a bug -- capital letters are not allowed as slugs. @ameyakhare can you TAL?

```suggestion
    "airplane:mongodb_findone",
```

Reply:
actually this is correct, builtins are special cased since the builtin namespace is `mongodb` and the function is `findOne` (capital o)

Reply:
Hmm, ideally this is a valid slug. Looks like only MongoDB has non-slug names. What if we add "aliases" here for each of those? https://github.com/airplanedev/airport/blob/8573361961ef526398947b834ae0e3493484a603/pkg/stdapi/mongodb/mongodb.go#L10-L24

e.g.

```go
		"findOneAndUpdate":  func() function.FunctionHandler { return handlers.FindOneAndUpdateHandler{} },
		"find_one_and_update":  func() function.FunctionHandler { return handlers.FindOneAndUpdateHandler{} },
```